src/state/user/hooks.tsx

```diff
@@ -57,7 +57,7 @@ export function useIsDarkMode(): boolean {
     shallowEqual
   )
 
-  return userDarkMode === null ? matchesDarkMode : userDarkMode
+  return false
```

Comment:
should there be a comment here explaining that this could be dynamic in the future to re-enable dark mode?

Reply:
Could add a ticket for dark mode... definitely post-MVP, but good to track these things early so we don't forget.

Reply:
yeah i'll create an issue for this too. choosing colors actually took way more time than i would have liked yesterday so i removed dark mode for now, but if we have time it'd be a fun task to enable it!